syntax = "proto3";

/* Message to be send by c# client to python client in order to process it. */
message CommandMessage 
{
    /* Contains two bytes. The first one is the byte for the Command and the second is the Variable to set byte. */
    bytes cmd_var_bytes = 1;
    string id = 2;
    float float_value = 3;
    double double_value = 4;
    /* Maybe extend it with int, sint uint e.t.c https://developers.google.com/protocol-buffers/docs/proto3 */

}
/* We may want to send multiple commands to python. Repeated means 0 or more times. */
message CommandList
{
    repeated CommandMessage messages = 1;
}